benchmarks/Microsoft.AspNetCore.Routing.Performance/Matching/DfaMatcherBuilderBenchmark.cs

```diff
@@ -0,0 +1,217 @@
+﻿using BenchmarkDotNet.Attributes;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing.Patterns;
+using Microsoft.AspNetCore.Routing.TestObjects;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Routing.Matching
+{
+    public class DfaMatcherBuilderBenchmark
```

Comment:
Can this be merged into https://github.com/aspnet/Routing/blob/release/2.2/benchmarks/Microsoft.AspNetCore.Routing.Performance/Matching/MatcherBuilderMultipleEntryBenchmark.cs

We have a set of benchmarks already test match creation. I think its OK to add variations, but if you think new tests are needed, I'd like it at least share the infrastructure.

Reply:
I wanted micro-benchmarks for these changes first and foremost for myself, but decided to include them in the PR anyway. That way it's easier for you to validate the changes.

I still have a few PRs pending (both submitted and in progress). Let's discuss the best approach later.

Reply:
Can these benchmarks be moved now?